pkgs/firehose/test_data/golden/comment_leaking.md

```diff
@@ -5,9 +5,11 @@
 
 The following packages contain symbols visible in the public API, but not exported by the library. Export these symbols or remove them from your publicly visible API.
 
-| Package | Leaked API symbols |
-| :--- | :--- |
-|package5|NonExported<br>NonExported2<br>TransitiveNonExported|
+| Package | Leaked API symbol | Leaking sources |
+| :--- | :--- | :--- |
+|package5|NonExported|package5_base.dart::Awesome::myClass|
```

Comment:
How come we have 3 lines here now instead of 1? New features for the leak detection? Add a PR description to explain.